webknossos/webknossos/dataset/_array.py

```diff
@@ -141,7 +141,9 @@ def __init__(self, path: Path):
 
     @classmethod
     def open(cls, path: Path) -> "WKWArray":
-        if (path / "header.wkw").is_file():
+        header_path = path / "header.wkw"
+
+        if header_path.exists() and header_path.is_file():
```

Comment:
This is a fix for a bug that @leowe found.

Reply:
One might argue that this is also a bug in UPath, but always sending two requests for the is_file check also seems wrong. In any case the fix is good atm!

Reply:
Yeah. I already have a fix for UPath in the pipeline.